docs/features/ulfm.rst

```diff
@@ -214,23 +144,38 @@ Running your application
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can launch your application with fault tolerance by simply using
-the normal Open MPI ``mpiexec`` launcher, with the
-``--with-ft ulfm`` CLI option:
+the normal Open MPI ``mpirun`` launcher, with the
+``--with-ft ulfm`` CLI option (or its synonym ``--with-ft mpi``):
 
 .. code-block::
 
    shell$ mpirun --with-ft ulfm ...
 
+  .. important:: By default, fault tolerance is not active at run time.
```

Comment:
```suggestion
   .. important: By default, fault tolerance is not active at run time.
```

Reply:
This change doesn't look right, what is the defect it tries to fix?

Reply:
I'm not sure what I was thinking here.  Let's disregard this comment...